### PR TITLE
Clarify NuGet SDK vs NuGet API

### PR DESCRIPTION
**Pull request by loic-sharma**
Addresses https://github.com/NuGet/docs.microsoft.com-nuget/issues/1400

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[103ab47](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/103ab471f7867f22f00dae5ca41b2e098e0d13b4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904172255016842-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8EOPXhDHjK5fZZ8fiKiFRFjs8LvzkCDiGdJqvpvKgBQ%3D&st=2019-04-17T22%3A51%3A12Z&se=2019-05-18T22%3A56%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/NuGet-Client-SDK.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ ms.topic: conceptual
...
-> [!Note]
-> Not to be confused with the [NuGet *Web* API](https://docs.microsoft.com/en-us/nuget/api/overview)
-
-The *NuGet Client SDK* refers to a group of .NET libraries centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [Nuget.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the earlier [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library.
+The *NuGet Client SDK* refers to a group of NuGet packages centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the earlier [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library.
 
 We are working on having a stable surface area that we can document soon.
 
+> [!Note]
+>  For documentation on the NuGet protocol, please refer to the [NuGet Server API](https://docs.microsoft.com/en-us/nuget/api/overview).
```
Comment (nkolev92) [inline docs/reference/NuGet-Client-SDK.md]: I would say NuGet server protocol. 

The client has a NuGet.Protocol library which it should not be confused with.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[cdbf9ed](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/cdbf9edf5c5fd2d9075d6a96801b319881908b03)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904172255560046-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4FO8ghDX%2F51hGgkGAZDjiyV1X1bPQGE%2BLHxo1Rk56E8%3D&st=2019-04-17T22%3A52%3A24Z&se=2019-05-18T22%3A57%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/NuGet-Client-SDK.md (hunk 1 above)
Comment (zivkan) [reply docs/reference/NuGet-Client-SDK.md]: Another alternative is the HTTP NuGet protocol?

Comment (zivkan) [inline docs/reference/NuGet-Client-SDK.md]: Use relative links to docs in the same docset: https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-the-same-docset

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2138aea](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/2138aea4589896b6225d00bab6c453e29f8d9d1c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904191526293082-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=guL68QJVWMr3HzS2bjX60v9MOxP5C7cd3WKm6ihqCzE%3D&st=2019-04-19T15%3A22%3A46Z&se=2019-05-20T15%3A27%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[9705e28](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/9705e28b2493093c918a56f8f4cfbe2859114a19)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) | [Details](#user-content-9fec4de6-895b050a)

<a id="9fec4de6-895b050a"></a>
### [docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/nuget/api/overview).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904191527464012-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=uNNrz%2B%2FxpVVECyEpRcQqouxysQe2Cr87k2R7wKKwBL0%3D&st=2019-04-19T15%3A24%3A00Z&se=2019-05-20T15%3A29%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5769950](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/576995035d33d839eb8649e219e5f2563106290b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) | [Details](#user-content-9fec4de6-8f8a8ea2)

<a id="9fec4de6-8f8a8ea2"></a>
### [docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/api/overview).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904191539017430-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LGQlNIs%2Bal0hTKYEMM79USsERzKfSsfYqVq8uvvyIsk%3D&st=2019-04-19T15%3A35%3A19Z&se=2019-05-20T15%3A40%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[3aaf6cd](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/3aaf6cd7ccb767d8ba196b5ffb03ba5d0e723495)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904191553451596-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0WU%2Fb0pniV3Fg%2FphHs4hxDMjALGz%2FLh9d2Ubnt5NczE%3D&st=2019-04-19T15%3A49%3A50Z&se=2019-05-20T15%3A54%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/NuGet-Client-SDK.md (hunk 1 above)
Comment (loic-sharma) [reply docs/reference/NuGet-Client-SDK.md]: Fixed

Comment (loic-sharma) [reply docs/reference/NuGet-Client-SDK.md]: Changed to NuGet server protocol

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[165e6cb](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/165e6cb1ecf01abaa630a350ae8b74d64f52e6fe)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) | [Details](#user-content-9fec4de6-4c822fc7)

<a id="9fec4de6-4c822fc7"></a>
### [docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/api/overview).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904191614510293-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RO0bfMXiz2XZlOoy%2Fqi%2Bcng3WFTPRaoOJBYw6F%2BcukM%3D&st=2019-04-19T16%3A10%3A53Z&se=2019-05-20T16%3A15%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[eadac4b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1409/commits/eadac4b2f30ccedaa366dbb0c21fa7e4f6745676)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/NuGet-Client-SDK.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-8/docs/reference/NuGet-Client-SDK.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-1409) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904191616249050-1409%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mkyXec%2B8%2FZR5HpjugQWioLk6ek1sAHYsAG0a%2BTenjj0%3D&st=2019-04-19T16%3A12%3A47Z&se=2019-05-20T16%3A17%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**